src/TodoList.js

```diff
@@ -1,18 +1,18 @@
 import React from "react";
 import TodoListItem from "./TodoListItem";
 
-const toDoList = [
+const todoList = [
     {id: 1, title: "Wake up"},
     {id: 2, title: "Drink water"},
     {id: 3, title: "Brush teeth"}
   ];
 
-function TodoList() {
+function TodoList(props) {
     return (
-        <div style={{ textAlign: 'left', backgroundColor: "aquamarine" }}>
+        <div style={{ textAlign: 'left'}}>
             <ul>
-                {toDoList.map(function (todo) {
-                return <TodoListItem key={todo.id} todo={todo} />;
+                {props.todoList.map((todo) => {
```

Comment:
This is not a make it or break it but it may be easier for you to use inline return here. It would be easier to read. I remember being so confused between inline and adding return. If you need help understanding it, feel free to reach out. 

```suggestion
                {todoList.map(todo => <ToDoListItem key={todo.id} todo={todo}/>)}
```